misc/diagram-util/build.gradle

```diff
@@ -64,3 +54,8 @@ test {
         suites 'src/test/resources/testng.xml'
     }
 }
+
+///*this is to let gradle infer the locations of the modules*/
+//java {
```

Comment:
Can we remove this?

Reply:
Earlier also there are many lines has been commented. I'll remove them in later commits